docs/UserGuide.adoc

```diff
@@ -87,6 +87,48 @@ Removes 1 egg from the 1st recipe's list of ingredients.
 * `recipe 2 remove i/Milk` +
 Removes milk from the 2nd recipe's list of ingredients.
 
+==== Add a cooking step
```

Comment:
Maybe we can call it "step" instead  of "cooking step", because some recipes might not be for cooking?

Reply:
"preparation step"? i think step is pretty clear as well